src/HttpHeader.cc

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (C) 1996-2019 The Squid Software Foundation and contributors
+ * Copyright (C) 1996-2020 The Squid Software Foundation and contributors
```

Comment:
An out of scope change?

```suggestion
 * Copyright (C) 1996-2019 The Squid Software Foundation and contributors
```